lib/conjur/command/variables.rb

```diff
@@ -26,10 +25,10 @@ class Conjur::Command::Variables < Conjur::Command
     var.arg_name "id [value]"
     var.command :create do |c|
       c.arg_name "mime_type"
-      c.flag [:m, :"mime-type"], default_value: "text/plain"
+      c.flag [:m, :"mime-type"]
 
       c.arg_name "kind"
-      c.flag [:k, :"kind"], default_value: "secret"
+      c.flag [:k, :"kind"]
```

Comment:
we'll have to figure out a different way to do this. interactive mode shouldn't affect non-interactive use of the CLI. otherwise we're doing to have empty values going into conjur